core/src/main/scala/com/salesforce/op/stages/impl/regression/OpXGBoostRegressor.scala

```diff
@@ -361,4 +364,18 @@ class OpXGBoostRegressionModel
   ttov: TypeTag[Prediction#Value]
 ) extends OpPredictionModel[XGBoostRegressionModel](
   sparkModel = sparkModel, uid = uid, operationName = operationName
-)
+) {
+  import OpXGBoost._
+  @transient private lazy val localModel = getLocalMlStage().map(s => s.model.asInstanceOf[MleapXGBoostRegressionModel])
+  @transient lazy protected val predict: Vector => Double = {
+    getSparkMlStage().map(s => s.predict(_))
+      .getOrElse{
+        features: Vector => {
+          val model = localModel.getOrElse(throw new RuntimeException(s"Could not find the wrapped Spark stage."))
+          val dm = new DMatrix(processMissingValues(Iterator(features.asXGB), 0.0F))
```

Comment:
nit: one-line doc about what pre-processing happens here and why would be nice. Maybe extract a method for it?